README.md

```diff
@@ -21,16 +21,17 @@ Mais finalement, c'est aussi plusieurs salons où l'on peut discuter librement e
 # Critères pour entrer sur le discord : 
 
 ```javascript
-const user = process.connectedUser(); 
+const me = ESCommunity.currentUser();
```

Comment:
Je verais bien un `you` plutôt que `me` et si on pouvait en profiter pour enlever les `;` 👼

Reply:
👍 j'aime l'idée du `you`
👎  moins l'idée du `;`